renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": ["config:base"],
+  "semanticPrefix": "build(all):",
+  "lockFileMaintenance": {"enabled": true},
+  "packageRules": [{"packagePatterns": ["^@types/"], "automerge": true}]
```

Comment:
Do we risk upgrading to the typings for the next major of a dependency we didn't upgrade yet without noticing when doint this?

Reply:
Had the same question, but the build task should catch this. On the other hand, I couldn't find whether renovate will wait for the CI to finish before merging. Also, see https://github.com/sinnerschrader/feature-hub/pull/44#discussion_r236404346

Reply:
Ok i think we should deactivate automerge for now. i will make another commit.

Reply:
No, you are right. This shouldn't necessarily lead to build errors, if for example we're not consuming the breaking part of the updated library.

Reply:
It does (from my experience) - but there could be use cases where we don't use apis that broke but then introduce access later and ts doesn't catch it anymore because of the typings mismatch (though ideally unit-test should still catch)